charts/artifact-caching-proxy/values.yaml

```diff
@@ -58,10 +58,12 @@ resources: {}
 #   cpu: 100m
 #   memory: 128Mi
 
-# log collection with Datadog
-logCollection:
-  enabled: true
-  config: '[{"source":"nginx","service":"artifact-caching-proxy","tag": "env:prod"}]'
+# Datadog Integration
+datadog:
+  metricsCollection:
+    enabled: true
```

Comment:
Is there a case where we don't want to enable metrics collection?
Initially I've put a logCollection.enabled as by default it's disabled (cf datadog doc), but it's not the case for metrics.

Reply:
I don't see a direct case indeed, as much as I don't see why log collection should be disabled here: datadog documentation states that log collection is disabled by default_on the datadog agent_. Switching the flag on/off in this chart does not impact the datadog agent configuration.

That's why I chose to have the same feature flag for both: we might want to disable it in the future.

(I though about cleaning up annotations if needed, due to an admission controller. But for statefulset/deployment I don't see the case, compared to ingress annotation that could be refused by the ingress admission)